openedx/core/djangoapps/xblock/runtime/learning_core_runtime.py

```diff
@@ -230,6 +233,40 @@ def get_block(self, usage_key, for_parent=None, *, version: int | LatestVersion
 
         return block
 
+    def get_block_assets(self, block):
+        """
+        Return a list of StaticFile entries.
+
+        TODO: When we want to copy a whole Section at a time, doing these
+        lookups one by one is going to get slow. At some point we're going to
+        want something to look up a bunch of blocks at once.
+        """
+        component_version = self._get_component_version_from_block(block)
+
+        # cvc = the ComponentVersionContent through-table
+        cvc_set = (
+            component_version
+            .componentversioncontent_set
+            .filter(content__has_file=True)
+            .order_by('key')
+            .select_related('content')
+        )
+
+        return [
+            StaticFile(
+                name=cvc.key,
+                url=reverse(
```

Comment:
Should this also be absolute, like `_lookup_asset_url` returns?

Reply:
You know, I think to date it returns a site-relative URL, but I think it makes more sense to return an absolute one, so I'll just do that and see if anything breaks. 😛